ansible/playbooks/files/Jenkinsfile

```diff
@@ -0,0 +1,22 @@
+#!groovy
+
+pipeline {
+  agent any
+
+  stages {
+    stage('Setup') {
+      steps {
+        git 'https://github.com/ansible/tower-example.git'
+        sh 'ansible --version'
+      }
+    }
+
+    stage('Provision') {
+      steps {
+        ansiColor('xterm') {
+          ansiblePlaybook playbook: 'helloworld.yml', colorized: true
```

Comment:
This won't work, right?

Reply:
I pulled this file in from the gsa.jenkins role, wasn't clear on the purpose except that it seemed to add things to jenkins itself for the base/example install. (meekly shuffles feet)